layers/meta-balena-fsl-arm/recipes-bsp/u-boot/files/bootscript-n8mm-allow-flashing-balenaOS.patch

```diff
@@ -1,10 +1,10 @@
-From 4e97f342a4dd8bcafdb887e192c7b27e84445e8e Mon Sep 17 00:00:00 2001
+From 7bd0d34430401d3d97cef889d4fd759e0b2250c2 Mon Sep 17 00:00:00 2001
```

Comment:
I usually remove this line from the patches as it's a local sha1 that adds no information and always changes.